tools/stonith_admin.c

```diff
@@ -528,6 +528,12 @@ main(int argc, char **argv)
         target = options.unregister_level;
     }
 
+    if (options.timeout > INT_MAX / 1000 || options.timeout < INT_MIN / 1000) {
```

Comment:
This is actually passed to functions as an `unsigned int` argument, so we should check

```
((options.timeout > (UINT_MAX / 1000)) || (options.timeout < 0))
```